src/main/java/duke/SavedTaskHandler.java

```diff
@@ -0,0 +1,117 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.function.Consumer;
+/**
+ * A class that encapsulates the SavedTaskHandler object
+ * which deals with loading tasks from the file and saving tasks in the file
+ *
+ * @author  Wee Xin Yang, Markus
+ * @version 0.1
+ * @since   2022-8-24
+ */
+public class SavedTaskHandler {
+
+    private final String filePath = "./data/saved.txt";
```

Comment:
Would this be better ? 🤔 
```
private static final String CURRENT_DIRECTORY = "user.dir";
Paths.get(System.getProperty(CURRENT_DIRECTORY), "data", "saved.txt").toString();
```
I guess this way would handle potential path related exceptions caused in some operating systems.